sonar-plugin-api/src/main/java/org/sonar/api/platform/Server.java

```diff
@@ -117,6 +117,8 @@
   /**
    * @return the server URL
    * @since since 2.4 on batch side only, since 5.6 on both batch side and server side (WebServer and Compute Engine)
+   * @deprecated since 6.5, please use {@link #getPublicRootUrl()} instead
    */
+  @Deprecated
   public abstract String getURL();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)